cmd/snap/cmd_known.go

```diff
@@ -31,11 +31,25 @@ import (
 	"github.com/jessevdk/go-flags"
 )
 
+type assertTypeName string
+
+func (n assertTypeName) Complete(match string) []flags.Completion {
+	names := asserts.TypeNames()
```

Comment:
well, `snap` itself is already importing `asserts` so no real need, but I included the REST endpoint for completeness.